packages/react/popper/src/Popper.tsx

```diff
@@ -63,6 +63,9 @@ const Popper = forwardRefWithAs<typeof POPPER_DEFAULT_TAG, PopperOwnProps>(
 
     const composedPopperRef = useComposedRefs(forwardedRef, popperRef);
 
+    const windowSize = useWindowSize();
```

Comment:
In reality, simply doing `useWindowSize()` without using the return would have been enough to make this all work by simply relying on re-rendering and the fact that we were reading `window.innerWidth` and `window.innerHeight` in `popper`.

But I thought this makes a lot more sense to make it all much more explicit and also the setup opens the door to support other boundary areas than window pretty simply.